pkg/crd/known_types.go

```diff
@@ -113,6 +111,52 @@ var KnownPackages = map[string]PackageOverride{
 	},
 }
 
+// ObjectMetaPackages overrides the ObjectMeta in all types
+var ObjectMetaPackages = map[string]PackageOverride{
+	"k8s.io/apimachinery/pkg/apis/meta/v1": func(p *Parser, pkg *loader.Package) {
+		// execute the KnowPackages for `k8s.io/apimachinery/pkg/apis/meta/v1` if any
+		if f, ok := KnownPackages["k8s.io/apimachinery/pkg/apis/meta/v1"]; ok {
+			f(p, pkg)
+		}
+		// This is a allow-listed set of properties of ObjectMeta, other runtime properties are not part of this list
```

Comment:
as per the meeting today, please link to sttts's comment here

Reply:
To clarify, I think @DirectXMan12 is referring to this comment https://github.com/kubernetes-sigs/controller-tools/pull/395#issuecomment-691919433. Can you confirm?

Reply:
yeah, that's what I was talking about.

Reply:
@DirectXMan12 @coderanger I'll update the PR

Reply:
done, rebased, squashed and comment added.